shopify/utils/utilities.py

```diff
@@ -0,0 +1,64 @@
+from .exceptions import *
+
+import jwt
+import re
+import sys
+
+if sys.version_info[0] < 3:  # Backwards compatibility for python < v3.0.0
+    from urlparse import urljoin
+else:
+    from urllib.parse import urljoin
+
+
+class SessionTokenUtility:
```

Comment:
One difference between Python and Ruby is that you can declare functions in Python. I'd be interested in hearing from other Python folks, but my feeling is that if the class doesn't have any instance methods, it doesn't need to be a class. The contents of this class could become module-level constants and public and private functions. I don't think Python has any concept of "private" functions in terms of blocking access, but the naming convention makes the intention clear, which is sufficient IMO

Reply:
So we can have something like a `session_token_utility.py` module that contains the necessary functions?